This patch makes "clang -print-multi-os-directory" print "." on amd64, which is
required by certain ports.

Introduced here: http://svnweb.freebsd.org/changeset/base/213492

Index: tools/clang/lib/Driver/Driver.cpp
===================================================================
--- tools/clang/lib/Driver/Driver.cpp
+++ tools/clang/lib/Driver/Driver.cpp
@@ -780,7 +780,7 @@ bool Driver::HandleImmediateArgs(const Compilation
       break;
 
     case llvm::Triple::x86_64:
-      llvm::outs() << "x86_64" << "\n";
+      llvm::outs() << "." << "\n";
       break;
 
     case llvm::Triple::ppc64:
